tests/pytests/test_kaguya_drivers.py

```diff
@@ -33,9 +33,13 @@ def test_kernels():
         for kern in kern_list:
             os.remove(kern)
 
-@pytest.mark.parametrize("label_type", ['pds3'])
-@pytest.mark.parametrize("image", image_dict.keys())
```

Comment:
There should be two parameterize sections here. I think we still need the image, image_dict.keys parameterize here

Reply:
This will cause combinations of (pds3, isis3) * (TC1S2B0_01_06691S820E0465, MNA_2B2_01_04192S136E3573), but there is no ISIS driver for kaguyatc.   Do we still want to make this change?

Reply:
Fair, lets keep this disabled until we get the kaguyatc driver in